db/schema.rb

```diff
@@ -974,7 +974,7 @@
     t.boolean  "email_digest",                              default: true
     t.boolean  "email_on_direct_message",                   default: true
     t.boolean  "official_position_badge",                   default: false
-    t.datetime "password_changed_at",                       default: '2017-09-06 18:19:39', null: false
+    t.datetime "password_changed_at",                       default: '2017-06-22 11:21:30', null: false
```

Comment:
Hey @iagirre this shouldn't go into any PR, it's a problem we've because of https://github.com/consul/consul/blob/master/db/migrate/20161102133838_default_password_changed_at.rb#L3 but haven't stopped yet to fix it

Reply:
Is one of those things that we just dodge normally instead of fix... but now it is fixed forever :) https://github.com/consul/consul/commit/e3a45baa37b60e49a0bc349f1609d7931248832a